### PR TITLE
Revoke PIL

### PR DESCRIPTION
**Pull request by joefitter**
* Added revoke routes inc confirm and success
* Added content for revocation tasks
* Added content for some success messages

### CODE REVIEW

File: pages/pil/read/views/index.jsx [hunk 1]
```diff
@@ -94,7 +94,14 @@ const PIL = ({ pil, profile, canUpdate, allowedActions, openTask }) => {
 
       <p className="control-panel">
         {
-          canUpdate && <Link page="pil.update" className="govuk-button" label="Amend licence"/>
+          canUpdate && (
+            <Fragment>
+              <Link page="pil.update" className="govuk-button" label={pil.status === 'active' ? 'Amend licence' : 'Reapply for licence'}/>
+              {
+                pil.status === 'active' && <Link page="pil.revoke.base" label="Revoke licence" />
```
Comment (wheelsandcogs) [inline pages/pil/read/views/index.jsx]: Should these labels be in snippets?

File: pages/task/content/status.js [hunk 2]
```diff
@@ -77,46 +82,54 @@ module.exports = {
...
       application: 'Reject application',
-      amendment: 'Reject amendment'
+      amendment: 'Reject amendment',
+      revocation: 'Reject revocation'
     },
     hint: {
       application: 'The applicant will need to create a new application to apply for this type of licence in the future.',
-      amendment: 'The applicant will need to create a new amendment to submit these changes again.'
+      amendment: 'The applicant will need to create a new amendment to submit these changes again.',
+      revocation: 'The applicant will need to create a new revocation for this licence to be revoked.'
```
Comment (wheelsandcogs) [inline pages/task/content/status.js]: There's potential for deduping a lot of the content in this file, but meh...

File: pages/common/content/success-messages.js [hunk 3]
```diff
@@ -38,6 +38,18 @@ module.exports = {
     }
   },
   pil: {
+    revocation: {
+      resubmitted: {
```
Comment (wheelsandcogs) [inline pages/common/content/success-messages.js]: `resubmitted`?

File: pages/task/content/status.js (hunk 2 above)
Comment (joefitter) [reply pages/task/content/status.js]: this is by choice - the simpler the file is, the more Phil can own it

File: pages/common/content/success-messages.js (hunk 3 above)
Comment (joefitter) [reply pages/common/content/success-messages.js]: yeah, resubmitted === submitted when it comes to messaging - we have content for resubmitted and not submitted, it avoids repeating them all for submitted

Comment (joefitter) [reply pages/common/content/success-messages.js]: it should definitely be the other way round but it isn't, and it's used _everywhere_ and.. oh my fuck budget just ran out

File: pages/pil/read/views/index.jsx (hunk 1 above)
Comment (joefitter) [reply pages/pil/read/views/index.jsx]: I was being lazy. I'll do it properly x

**Pull request merged**